src/drivers/mct/cime_config/config_component.xml

```diff
@@ -1804,6 +1804,17 @@
     </desc>
   </entry>
 
+  <entry id="NODE_FAIL_REGEX">
+    <type>char</type>
+    <default_value>UNSET</default_value>
```

Comment:
Good point. This break my code since I check for None, not UNSET. If I leave this blank, do I get a python None if I ask for it and the machine doesn't specify this value?